app/src/lib/git/tag.ts

```diff
@@ -0,0 +1,34 @@
+import { git } from './core'
+import { Repository } from '../../models/repository'
+
+/**
+ * Create a new tag on the given target commit.
+ *
+ * @param repository        - The repository in which to create the new tag.
+ * @param name              - The name of the new tag.
+ * @param targetCommitSha   - The SHA of the commit where the new tag will live on.
+ */
+export async function createTag(
+  repository: Repository,
+  name: string,
+  targetCommitSha: string
+): Promise<void> {
+  const args = ['tag', '-a', '-m', '', name, targetCommitSha]
+
+  await git(args, repository.path, 'createTag')
+}
+
+/**
+ * Gets all the local tags.
+ *
+ * @param repository    The repository in which to get all the tags from.
+ */
+export async function getAllTags(
+  repository: Repository
+): Promise<ReadonlyArray<string>> {
+  const args = ['tag']
+
+  const tags = await git(args, repository.path, 'getAllTags')
+
+  return tags.stdout.split('\n').filter(Boolean)
```

Comment:
@rafeca is the `.filter(Boolean)` to filter out empty lines? we generally avoid boolean coercion in the codebase, so maybe this could be: 
```suggestion
  return tags.stdout.split('\n').filter(s => s !== '')
```

Reply:
Yes, it's to filter empty lines. Thanks for the suggestion!